### ISSUE TITLE
Improve / consolidate / condense information displayed on users#index

### ISSUE THREAD

**Issue by carpodaster** [rails-girls-summer-of-code/rgsoc-teams#83]
- [x] Remove common prefixes from `interested_in`'s human readable form (e.g. "Helping as a: coach, mentor, remote coach") and transform it into a nice + short text block
- [x] Display cityand location as "Germany/Buxtehude" (depends on #82)
- [x] Add company name? Should also have a custom link if we add this.

Related question: what shall we do with `users#show`? Right now it's a bit of an unloved stepchild; there would be lots of space to display all the information we have trouble cramming into that user listing on `#index`.

@svenfuchs: does this reflect what we talked about in Slack yesterday?

/discuss

**Comment by klappradla** [comment]
Moving this into the [Community Page](https://github.com/rails-girls-summer-of-code/rgsoc-teams/milestone/13) milestone, since this is an integral part of the whole "community page" project 😉 

We will need some redefinitions for this though.

**Comment by branquinhoaa** [comment]
Hey guys, @carpodaster @klappradla 

obs.:
The page is no longer users#index, but community instead. 

I would like to confirm that you still want the same things. I also have some doubts about it. 
## Changes you guys want to do:

1. Remove "Interested In" field and replace it with a short text block (same column name?). That will include a small addition into users model (add an extra field for interests) and an addition into user#edit form too.

2. Instead of display two columns (country and city), we will display just one (column name can be 'location' ?) in the format: "Germany/Buxtehude"

3. Will we include company name?

**Comment by klappradla** [comment]
`1` and `3` still make sense I'd say (can you 👍/👎 @carpodaster?)
For `2`: I'm pretty sure this is only out how city + country should be displayed in the view, not about changes to the datamodel :v:

**Comment by carpodaster** [comment]
I would not swap the interested in for a text field. Maybe an _additional_ text field would make sense to give a bit more detail, but the predefined values for interested-in allow for better searching / categorizing (see also my comment on https://github.com/rails-girls-summer-of-code/rgsoc-teams/pull/849#pullrequestreview-61043457 about interested vs. coach role)

Other than that, I do not recall what I meant with the first item on the list :(

I would also assume that item no. 2 was about _displaying_ the info, not having to different columns. The column name "location" makes sense, yes :+1: 

I don't have strong feelings about the company name anymore. But since we have that "I represent a company", we might as well include it :+1:

**Comment by branquinhoaa** [comment]
@klappradla I'm sorry I did not express myself very well. The column I said is not the model column, but the column of the table displayed in the view.  I will start working on this issue :dancer:

**Comment by klappradla** [comment]
Hm, reading @carpodaster's comment above ☝️ 
I think it should look like this:
* we keep the `interested_in` string field with predefined values for better searchability (coach, etc.)
* we add an additional text for a short bio

I did not yet read up everything in your PR, but I think this should still be the way to go.

**Comment by juuh42dias** [comment]
@klappradla  "with predefined values for better searchability" with autocomplete?

**Comment by klappradla** [comment]
Hm, I'm not sure if we're talking about the same thing here @branquinhoaa 🤔 

In the current user edit form, picking the interests looks like this:
<img width="479" alt="screen shot 2017-09-13 at 18 52 18" src="https://user-images.githubusercontent.com/3491815/30389959-cf5a41a4-98b4-11e7-9c28-9b49e27c3adc.png">

**Comment by branquinhoaa** [comment]
@klappradla yeah, we will keep this info in the interest_in and add a short text that will explain better about the user interests. We are talking about the same thing... We are working on that and we will submit the PR soon. :+1:

**Comment by klappradla** [comment]
Ok, all good 💚 
Sorry, my brain is kinda slow today, craving sleep and cookies ✌️

**Comment by carpodaster** [comment]
Thanks for clarifying, @branquinhoaa. I, too, thought it would be replaced

**Comment by juuh42dias** [comment]
With aditional text

![rgsoc teams app 20](https://user-images.githubusercontent.com/1828204/30401492-d54ba87c-98b0-11e7-8f97-5c6555b006f7.png)
